src/clientiface.cpp

```diff
@@ -238,11 +239,9 @@ void RemoteClient::GetNextBlocks (
 
 			/*
 				Don't generate or send if not in sight
-				FIXME This only works if the client uses a small enough
-				FOV setting. The default of 72 degrees is fine.
+				NOTE This only works if client and server use the same
```

Comment:
Or you mean FIXME since it won't work otherwise? That'd be a fair point. Happy to change it back to FIXME.